src/graphql/queries/lambdas/sendNotifications.graphql

```diff
@@ -0,0 +1,53 @@
+query GetUserDetails_Sn($id: ID!) {
+  getProfile(id: $id) {
+    displayName
+    email
+    notificationSettings {
+      notificationTokens
+      enableEmail
+      enableMention
+      enableAssign
+    }
+  }
+}
+
+query GetDisplayName_Sn($id: ID!) {
+  getProfile(id: $id) {
+    displayName
+  }
+}
+
+query GetColonyName_Sn($id: ID!) {
+  getColony(id: $id) {
+    name
+    metadata {
+      displayName
+    }
+  }
+}
+
+query GetColonyContributors_Sn($colonyAddress: ID!) {
+  getContributorsByColony(colonyAddress: $colonyAddress) {
+    items {
+      notificationSettings {
+        enablePush
+        enableEmail
+        enableActions
+        enableDecisions
+      }
+      user {
+        profile {
+          id
+          displayName
+          email
+        }
+      }
+    }
+  }
+}
+
+mutation CreateNotification_Sn($input: CreateNotificationInput!) {
```

Comment:
Also, this is mutation, placed inside the "queries" folder

Reply:
This was a concious decision as I wanted to keep all the graphql queries for a lambda in one file, now that this file is moved to `amplify/backend/function/sendNotification/lib/sendNotifications.graphql` do you think it makes sense to keep queries and mutations together like this?